make/modules/java.base/Copy.gmk

```diff
@@ -45,7 +45,7 @@ endif
 
 ################################################################################
 # Copy the microsoft runtime libraries on windows
-ifeq ($(call isTargetOs, windows), true)
+ifeq ($(call isTargetOs, windows)+$(CREATING_BUILDJDK), true+false)
```

Comment:
Is the problem here that we don't have build versions of the runtime libraries so we simply trust that they are present on the build host instead?

Reply:
Yes. And this applies to the other fix above as well. Hm. Maybe it is enough to avoid copying them here, and then I don't need the patch in CreateJmods.gmk